rand_distr/src/normal.rs

```diff
@@ -145,14 +150,12 @@ pub struct LogNormal {
 impl LogNormal {
     /// Construct a new `LogNormal` distribution with the given mean
     /// and standard deviation.
-    ///
-    /// # Panics
-    ///
-    /// Panics if `std_dev < 0`.
     #[inline]
-    pub fn new(mean: f64, std_dev: f64) -> LogNormal {
-        assert!(std_dev >= 0.0, "LogNormal::new called with `std_dev` < 0");
-        LogNormal { norm: Normal::new(mean, std_dev) }
+    pub fn new(mean: f64, std_dev: f64) -> Result<LogNormal, Error> {
```

Comment:
Hmm, these parameter names are wrong; some conversion is needed if parametrising via mean and SD. In another application we decided to use [mean, CV](https://github.com/SwissTPH/openmalaria/blob/master/model/util/sampler.cpp#L101).

I'm not sure what the best approach is here; e.g. we could add `LogNormal::from_mean_CV`. We should still support parametrisation as in this function; recently I've been calling the parameters `mu, sigma` though that is still confusing.

Reply:
Could point. This is kind of orthogonal to this PR, however it means I would have to use different error types for `Normal` and `LogNormal`.

Reply:
I believe the parameter names are correct when talking about the mean and standard deviation of the logarithm of the distribution, I'll clarify the docs accordingly.

Reply:
You're right about this being orthogonal; moved to #775.